tests/test_piiremover.py

```diff
@@ -43,12 +43,37 @@ def test_pii_remover_catches_phone_numbers(self):
         self.assertTrue(pii_remover('This is not a real telephone number (03069) 990123!') == success)
         self.assertTrue(pii_remover('This is not a real telephone number 03069 990123!') == success)
 
+    def test_pii_remover_catches_credit_card_numbers(self): 
+            
+        '''
+        Source for fake card numbers: 
+        https://www.paypalobjects.com/en_US/vhelp/paypalmanager_help/credit_card_numbers.htm
+        '''
+
+        success = 'My fake credit card number is {{CREDIT CARD}}, thank you!'
+        
+        self.assertTrue(pii_remover('My fake credit card number is 378282246310005, thank you!') == success)           
```

Comment:
Unfortunately these conflict with the telephone number regexes. This is always going to happen, and short of using something really clever (like machine learning and natural language processing) I don't see an easy way of avoiding it.

I suppose we could change the tests to confirm whether or not the PII has been removed, regardless of what has been put in its place, but often this will result in incomplete matches, so only half the PII will be removed. 

I'm minded to keep the regexes as specific as possible to the particular categories of PII for now, not making any special effort to capture edge cases, and to keep an eye on the sort of things that are coming up. We can then address any overlapping matches when it becomes apparent that they are a problem.

Even if there is a partial match, it is easy to identify which surveys have _some_ PII, and the redaction can then be manually edited. It might be good to add an admin page to fix dodgy redaction, and manually redact false negatives that are identified by classifiers.

I've recorded a few of the false positives and false negatives that came up in the most recent round of classification in #42